src/popup/components/Modals/AccountCreate.vue

```diff
@@ -45,8 +53,13 @@
 <script lang="ts">
 import { defineComponent, PropType, ref } from 'vue';
 import { useStore } from 'vuex';
-import { MODAL_MULTISIG_VAULT_CREATE } from '@/constants';
+import {
+  MODAL_MULTISIG_VAULT_CREATE,
+  PROTOCOL_AETERNITY,
+  PROTOCOL_BITCOIN,
+} from '@/constants';
 import { useConnection, useModals } from '@/composables';
+import { Protocol } from '@/types';
```

Comment:
Please put this before all local imports and use `import type` to avoid dependency cycles.